cli/runtime/component/tty.go

```diff
@@ -0,0 +1,15 @@
+package component
+
+import (
+	"github.com/mattn/go-isatty"
+	"os"
+	"strings"
+)
+
+func IsTTYEnabled() bool {
+	ttyEnabled := false
+	if os.Getenv("TANZU_CLI_NO_COLOR") != "" || os.Getenv("NO_COLOR") != "" || strings.EqualFold(os.Getenv("TERM"), "DUMB") || !isatty.IsTerminal(os.Stdout.Fd()) {
+		ttyEnabled = true
```

Comment:
I'm not well versed in the concept of a `TTY`, but I think the enable/disable is reversed in this function.
To print colours I believe you need a tty.  So when we don't print colours, we probably want to return `false`.

Maybe we can clarify things by changing the name of the function and file?  I don't have a good suggestion though.  Maybe another reviewer can chime in?